packages/workflow/src/RoutingNode.ts

```diff
@@ -209,17 +211,29 @@ export class RoutingNode {
 
 				returnData.push(...responseData);
 			} catch (error) {
-				if (get(this.node, 'continueOnFail', false)) {
-					returnData.push({ json: {}, error: error.message });
+				interface RoutingError extends NodeError {
+					isAxiosError: boolean;
+					description: string | undefined;
+					response?: { status: number };
+				}
```

Comment:
Eventually we should straighten out the node error types. Hopefully good enough for now.